src/Appwrite/Platform/Tasks/Retry.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Utopia\CLI\Console;
+use Utopia\Platform\Action;
+use Utopia\Queue\Client;
+use Utopia\Queue\Connection;
+use Utopia\Validator\Text;
+
+class Retry extends Action
+{
+    public static function getName(): string
+    {
+        return 'retry';
+    }
+
+
+    public function __construct()
+    {
+        $this
+            ->desc('Retry Queue')
+            ->param('name', '', new Text(128), 'Queue name')
+            ->inject('queue')
+            ->callback(fn ($queueName, $queue) => $this->action($queueName, $queue));
+    }
+
+    public function action(string $queueName, Connection $queue): void
+    {
+        $queueClient = new Client($queueName, $queue);
+
+        if ($queueClient->sumFailedJobs() === 0) {
```

Comment:
I might call this method `countFailedJobs `

Reply:
This would need to be updated in the Queue lib

Reply:
I've updated all sumX methods to countX methods